.github/workflows/update-schema.yaml

```diff
@@ -36,37 +36,28 @@ jobs:
           echo "CACHE_DURATION = 0" >> config/settings/base.py
           echo "AGGREGATE_COUNT_THRESHOLD = 5" >> config/settings/base.py
           python manage.py export_openapi_schema --api chord_metadata_service.mohpackets.apis.core.api | python -m json.tool > chord_metadata_service/mohpackets/docs/schema.json
-      
-      - name: Commit new schema.json
-        uses: stefanzweifel/git-auto-commit-action@v5
-        id: auto-commit-action
-        with:
-          commit_message: Update schema.json
-          file_pattern: 'chord_metadata_service/mohpackets/docs/schema.json'
-
-      - name: Update schema.json with new SHA
-        if: steps.auto-commit-action.outputs.changes_detected == 'true'
-        run: |
-          REPO_NAME=${{ github.repository }}
-          SHA=${{ steps.auto-commit-action.outputs.commit_hash }}
-          SCHEMA_PATH=chord_metadata_service/mohpackets/docs/schema.json
-          sed -i 's|"description": "This is the RESTful API for the MoH Service."|"description": "This is the RESTful API for the MoH Service. Based on https://raw.githubusercontent.com/'"$REPO_NAME"'/'"$SHA"'/'"$SCHEMA_PATH"'"|' $SCHEMA_PATH
 
       - name: Install widdershins
         run: npm install -g widdershins
 
       - name: Convert schema to OpenAPI documentation
         run: |
           npx widdershins ./chord_metadata_service/mohpackets/docs/schema.json -o ./chord_metadata_service/mohpackets/docs/schema.md -u ./chord_metadata_service/mohpackets/docs/widdershins/templates/openapi3 -c true --omitHeader true
-      
+
       - name: Install PyYAML
         run: pip install PyYAML
 
       - name: Convert schema.json to schema.yml
         run: python -c 'import json, yaml; json.load(open("chord_metadata_service/mohpackets/docs/schema.json")); print(yaml.dump(json.load(open("chord_metadata_service/mohpackets/docs/schema.json"))))' > chord_metadata_service/mohpackets/docs/schema.yml
 
-      - name: Update schemas with SHA
+      - name: Increment schema version
```

Comment:
Is there a reason we can't use the SHA of the updated schema.yml (or schema.json, whichever) document as the schema version?

Reply:
The problem is the current branch SHA will get deleted once it merges. Sure I can put the SHA in, but if someone were to go back and find it, they won't be able to.

Reply:
unless we want to keep all the branch PR around

Reply:
Not the branch SHA, the file sha. That stays around.

Reply:
For example, in this deleted branch https://github.com/CanDIG/htsget_app/blob/9d00ade5811bf48727b7b73516c30d06796d5f81/tests/test_htsget_server.py and in the stable branch it was merged into https://github.com/CanDIG/htsget_app/blob/stable/tests/test_htsget_server.py, the sha for test_htsget_server.py is 6429207.

Reply:
Sure, I can try that

Reply:
maybe that's not the file sha, I mean the commit sha.